proto/src/prost/v0_34/tendermint.rpc.grpc.rs

```diff
@@ -18,3 +18,194 @@ pub struct ResponseBroadcastTx {
     #[prost(message, optional, tag = "2")]
     pub deliver_tx: ::core::option::Option<super::super::abci::ResponseDeliverTx>,
 }
+/// Generated server implementations.
```

Comment:
If possible, avoid generating bindings for this package. It was removed from CometBFT as per https://github.com/cometbft/cometbft/issues/650